README.md

```diff
@@ -18,18 +18,37 @@ The Vantage MCP Server is an open-source tool, written in Golang, that lets you
 
 The Vantage MCP Server currently exposes the following tools, which can be invoked by any compatible MCP client (e.g., Claude, Cursor, Goose):
 
-- `get-costs`
+- `query-costs`
+  - A general purpose way to fetch cost data using VQL.
 
-  - Allows filtering costs by date and VQL.
+- `list-costs`
+
+  - Display all the costs in an associated Cost Report.
 
 - `list-cost-reports`
 
   - List all cost reports available.
 
+- `get-cost-report-forecast`
+
+  - List all Forecasts of spending related to a Cost Report.
+
 - `list-cost-integrations`
 
   - List all cost provider integrations (e.g., AWS, Azure, GCP) available to provide costs data from and their associated accounts.
 
+- `list-cost-providers`
+  - List of just the providers that the given Workspace has shared with it.
```

Comment:
nit: Is there a style we want to conform to when documenting the entities to standardize on? seeing a mix of titlecase and none

Reply:
Skimming through docs.vantage.sh seems we upper case the entities, I'll do pass at making them all consistent.

Reply:
@conner updated